tests/arcus/ArcusCommunicationPrivateTest.cpp

```diff
@@ -155,18 +163,20 @@ TEST_F(ArcusCommunicationPrivateTest, ReadMeshGroupMessage)
 
     // - Load 'global' settings:
     std::unordered_map<std::string, std::string> raw_settings;
-    loadTestSettings("../tests/test_global_settings.txt", &mesh_message, &raw_settings);
+    loadTestSettings(std::filesystem::path(__FILE__).parent_path().parent_path().append("test_global_settings.txt").string(),
+                     &mesh_message,
+                     &raw_settings);
 
     // - Create mesh-message-mesh:
     cura::proto::Object* mesh = mesh_message.add_objects();
 
     // - - Read cube vertices from a test-file, then add to message:
-    std::ifstream cube_verts_file("../tests/cube_vertices.txt");
+    std::ifstream cube_verts_file(std::filesystem::path(__FILE__).parent_path().parent_path().append("cube_vertices.txt").string());
     ASSERT_TRUE(cube_verts_file.is_open());
 
     std::vector<float> raw_vertices;
 
-    float next;
+    float next{};
```

Comment:
This double-initialises the variable.

Reply:
honest question... and that is worse then an uninitialized variable (which just happen to take the value of the memory it was mapped to?

Will remove it for consistency, but I myself prefer to always initialize a value, reduces the changes on strange bugs